tests/python_package_test/test_sklearn.py

```diff
@@ -163,6 +163,50 @@ def test_dart(self):
         self.assertGreaterEqual(score, 0.8)
         self.assertLessEqual(score, 1.)
 
+    # sklearn < 0.22 does not have a stacking classifier
+    @unittest.skipIf(sk_version < '0.22.0', 'scikit-learn version is less than 0.22')
+    def test_stacking_classifier(self):
+        from sklearn.ensemble import StackingClassifier
+
+        X, y = load_iris(return_X_y=True)
+        X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+        classifiers = [('gbm1', lgb.LGBMClassifier()),
+                       ('gbm2', lgb.LGBMClassifier())]
+        clf = StackingClassifier(estimators=classifiers,
+                                 final_estimator=lgb.LGBMClassifier())
+        clf.fit(X_train, y_train).score(X_test, y_test)
+        self.assertEqual(clf.n_features_in_, 4)  # test number of input features
```

Comment:
Please add more `assert`s. Refer for example to GridSearchCV test
https://github.com/microsoft/LightGBM/blob/2e2757f183a17c2998c1002531fbbf27d0010783/tests/python_package_test/test_sklearn.py#L181-L187